cache_images/fedora-netavark_packaging.sh

```diff
@@ -51,6 +52,7 @@ INSTALL_PACKAGES=(\
     openssl-devel
     podman
     policycoreutils
+	protobuf-devel
```

Comment:
Is editor setup to follow the indentation style/method used in the rest of the file?

Reply:
evidently not